decidim-meetings/db/migrate/20210413050917_update_published_at_to_existing_meetings.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# By default all existing meetings were published when created
+# This migration prevents un-publishing all existing meetings
+class UpdatePublishedAtToExistingMeetings < ActiveRecord::Migration[5.2]
+  def change
+    Decidim::Meetings::Meeting.find_each do |meeting|
+      if meeting.published_at.nil?
+        # rubocop:disable Rails/SkipsModelValidations
+        # use update_column to prevent running callbacks
+        meeting.update_column :published_at, meeting.created_at
```

Comment:
💬 When loading a model inside a migration, we prefer to define locally the model class to avoid loading the real model code during its initialization, data validation or any other moment. That way, you don't need to use the `update_column` method. This can be done adding the following class before the `change` method and using the class `Meeting` instead of `Decidim::Meetings::Meeting`:

```ruby
class Meeting < ApplicationRecord
  self.table_name = :decidim_meetings_meetings
end
```

You can also check how it's done in other migrations of this module, as [here](https://github.com/decidim/decidim/blob/develop/decidim-meetings/db/migrate/20180809134748_add_upcoming_events_as_content_block.rb) or [here](https://github.com/decidim/decidim/blob/develop/decidim-meetings/db/migrate/20201016112641_add_registration_type_and_url_to_meetings.rb).

Reply:
BTW, there are some scenarios where this can't be done. For example when using PaperTrail to store records changes, as it would store a wrong name for the class (`Meeting` instead of `Decidim::Meetings::Meeting`).